src/h5web/vis-packs/core/raw/RawVis.tsx

```diff
@@ -1,9 +1,8 @@
 import type { ReactElement } from 'react';
 import styles from './RawVis.module.css';
-import type { HDF5Value } from '../../../providers/hdf5-models';
 
 interface Props {
-  value: HDF5Value;
+  value: unknown;
```

Comment:
I chose to use an explicit type here for consistency with the other visualizations.